lib/cli.js

```diff
@@ -29,6 +29,17 @@ const debug = require("debug")("eslint:cli");
 //------------------------------------------------------------------------------
 
 /**
+ * Predicate function for whether or not to apply fixes in quiet mode.
+ * If a message is a warning, do not apply a fix.
+ * @param {LintResult} lintResult The lint result.
+ * @returns {boolean} True if the lint message is an error (and thus should be
+ * autofixed), false otherwise.
+ */
+function quietFixPredicate(lintResult) {
+    return Boolean(lintResult && lintResult.severity === 2);
```

Comment:
Why can't this be `return lintResult.severity === 2`? Is `lintResult` ever null?